Docs: Fix dev doc 'documentation' (#1050)
hedgedoc/hedgedoc: docs/content/dev/documentation.md

# Documentation

Our documentation is build with [mkdocs][mkdocs].

## Writing

All documentation files are found in the `docs/content` directory of
the [hedgedoc/hedgedoc repo](https://github.com/hedgedoc/hedgedoc). These files are just normal markdown files with
nothing special about them.

The configuration for mkdocs lies in the `docs` folder in a file called `mkdocs.yml`. With that file the theme and menu - among others - can be configured.

>   **Please note:** Any new files need to be linked to by other files or put in the navigation, or the files will be very hard to find on the documentation website.

Reviewer: ```suggestion
  **Please note:** Any new files need to be linked to by other files or put in the navigation, otherwise the files will be very hard to find on the documentation website.
```

Author: fixed

## Deployment

The documentation is deployed with [mkdocs][mkdocs].

> The repo [docs.hedgedoc.org](https://github.com/hedgedoc/docs.hedgedoc.org) is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as that is latest release.

Reviewer 2: ```suggestion
The repository [docs.hedgedoc.org](https://github.com/hedgedoc/docs.hedgedoc.org) is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as that is latest release.
```

Author: fixed

> The repo [docs.hedgedoc.org repo](https://github.com/hedgedoc/docs.hedgedoc.org). is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as that is latest release.

Reviewer 3: ```suggestion
The repo [docs.hedgedoc.org](https://github.com/hedgedoc/docs.hedgedoc.org) is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as that is latest release.
```

Author: fixed

> The repository [docs.hedgedoc.org](https://github.com/hedgedoc/docs.hedgedoc.org) is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as that is latest release.

Reviewer: ```suggestion
The repository [docs.hedgedoc.org](https://github.com/hedgedoc/docs.hedgedoc.org) is used to deploy the actual website to github.io. Currently only the `master` branch is deployed as it contains the latest release.
```

Author: fixed
